README.md

```diff
@@ -1040,6 +1041,49 @@ scriptUnloaded = function () {
 
 ## Advanced Scripting
 
+### Libraries
+
+#### Third Party Libraries
+
+Loading of third party libraries is supported the same way as loading the openHAB JavaScript library:
+
+```javascript
+var myLibrary = require('my-library');
+```
+
+Run the `npm` command from the `automation/js` folder to install third party libraries (Node.js modules), e.g. from [npm](https://www.npmjs.com/search?q=openhab).
+This will create a `node_modules` folder (if it doesn't already exist) and install the library and it's dependencies there.
+
+There are already some openHAB specific libraries available on [npm](https://www.npmjs.com/search?q=openhab), you may also search the forum for details.
+
+#### Creating Your Own Library
+
+You can also create your own personal JavaScript library for openHAB, but you can not just create a folder in `node_modules` and put your library code in it!
+`npm` will remove everything from `node_modules` that has not been properly installed when it is run.
+
+You can follow these steps to create your own library (it's called a Node.js module):
+
+1. Create a separate folder for your library, you may also init a Git repository.
```

Comment:
Maybe it is worth mentioning, that this folder should not ne placed in `automation/js`. Otherwise the engine picks the `*.js` files from the library before they are installed into `node_modules`.

Reply:
@sfriedle I've added this, thanks.
Can you re-review and say if it looks good to you?